### PR TITLE
[IMP] pos_place : preserve place_id on invoiced orders

### PR DESCRIPTION
**Pull request by legalsylvain**
- [IMP] add demo data;
- [IMP] add place_id on account invoices
- [IMP] when invoicing via PoS, place_id is preserved on invoices

[GRAP-Task-424](https://erp.grap.coop/web#id=424&action=2148&active_id=4&model=project.task&view_type=form&menu_id=1673)

### CODE REVIEW

**Review by quentinDupont — APPROVED** [review]

File: pos_place/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2022 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
```
Comment (ivantodorovich) [inline pos_place/models/account_invoice.py]:
```suggestion
from odoo import fields, models
```

Travis is unhappy

```pos_place/models/account_invoice.py:4:1: F401 'odoo.api' imported but unused```

**Review by ivantodorovich — APPROVED** [review]
LG 👍🏻 just a comment about traivs

File: pos_place/models/account_invoice.py (hunk 1 above)
Comment (legalsylvain) [reply pos_place/models/account_invoice.py]: thanks. I fixed it.

**Review by flotho — APPROVED** [review]

**Comment by legalsylvain** [comment]
/ocabot merge patch 

Thanks for your reviews !

**Pull request merged**